src/storage/redis_metadata.h

```diff
@@ -245,4 +245,9 @@ class BloomChainMetadata : public Metadata {
   ///
   /// @return the total capacity value
   uint32_t GetCapacity() const;
+
+  /// Check the bloom chain is scaling or not
```

Comment:
I think it's needless to comment on simple methods such as `IsScaling` or `GetCapacity` - they have self-explanatory names. BTW, the implementation of `IsScaling` is so short/simple that you can write it in the header file.